tools/sony9pin/devices.cpp

```diff
@@ -10,18 +10,305 @@
+  { 0xd011, { "Solid State Logic", "9000j" } },
+  { 0xd012, { "Solid State Logic", "Axiom" } },
+  { 0xd013, { "Solid State Logic", "APS" } },
+  { 0xd014, { "Solid State Logic", "Scenaria" } },
+  { 0xd015, { "Solid State Logic", "Omnimix" } },
+  { 0xd016, { "Solid State Logic", "VisionTrk" } },
+  { 0xd017, { "Solid State Logic", "Aysis Air" } },
+  { 0xd018, { "Solid State Logic", "Avant" } },
+  { 0xd019, { "Solid State Logic", "Axiom MT" } },
+  { 0xd030, { "SONY", "MAV-1000" } },
+  { 0xd031, { "SONY", "MAV-1200" } },
+  { 0xd035, { "SONY", "MAV-2000" } },
+  { 0xd036, { "SONY", "BKMA-2010" } },
+  { 0xd040, { "SONY", "DME-9000" } },
+  { 0xd050, { "", "DADR-5000" } },
+  { 0xd051, { "", "DADR-5032" } },
+  { 0xd052, { "", "DADR-5048" } },
+  { 0xd053, { "", "DADR-5064" } },
+  { 0xd081, { "DPS", "DPS-PVR" } },
+  { 0xd090, { "", "DN-C680" } },
+  { 0xd091, { "", "DN-M51050R" } },
+  { 0xd0a0, { "AVID", "AS 4" } },
+  { 0xd0a1, { "AVID", "AS 8" } },
+  { 0xd0a2, { "AVID", "AS16" } },
+  { 0xd0a3, { "AVID", "AS32" } },
+  { 0xd0a8, { "AVID", "AV 4" } },
+  { 0xd0a9, { "AVID", "AV 8" } },
+  { 0xd0aa, { "AVID", "AV16" } },
+  { 0xd0ab, { "AVID", "AV32" } },
+  { 0xd0b0, { "", "RADAR" } },
+  { 0xd0c0, { "Genex", "GX2000" } },
+  { 0xd0c1, { "Genex", "GX8000" } },
+  { 0xd0c2, { "Genex", "GX8016" } },
+  { 0xd0c3, { "Genex", "GX8024" } },
+  { 0xd0c4, { "Genex", "GX8032" } },
+  { 0xd0c5, { "Genex", "GX8040" } },
+  { 0xd0c6, { "Genex", "GX8048" } },
+  { 0xd0c7, { "Genex", "GX8056" } },
+  { 0xd0c8, { "Genex", "GX8064" } },
+  { 0xd0d0, { "", "Dolby Drv" } },
+  { 0xd0d1, { "", "Dolby Drv" } },
+  { 0xd0d4, { "", "dAVE SP" } },
+  { 0xd0e0, { "", "Protools" } },
+  { 0xd0e1, { "", "Protools" } },
+  { 0xd0e2, { "", "Protools" } },
+  { 0xd0f3, { "", "Zeta 3" } },
+  { 0xd0f8, { "CB Electronics", "MC-1" } },
+  { 0xd0f9, { "", "LANC I/F" } },
+  { 0xf000, { "JVC", "CR850" } },
+  { 0xf002, { "Bosch", "BCN-52" } },
+  { 0xf003, { "Abekas", "AbekasA62" } },
+  { 0xf004, { "Abekas", "AbekasA64" } },
+  { 0xf005, { "Abekas", "AbekasA60" } },
+  { 0xf006, { "Abekas", "AbekasA53D" } },
+  { 0xf007, { "Ampex", "VPR6" } },
+  { 0xf008, { "Abekas", "AbekasA57" } },
+  { 0xf009, { "", "DCR100" } },
+  { 0xf00a, { "", "KR-M800" } },
+  { 0xf00b, { "", "SA-F911" } },
+  { 0xf00c, { "Hitachi", "VL-D500" } },
+  { 0xf00d, { "Hitachi", "VL-D505" } },
+  { 0xf00e, { "Hitachi", "VL-S100" } },
+  { 0xf00f, { "", "FOSTEX" } },
+  { 0xf010, { "", "D-20" } },
+  { 0xf011, { "Abekas", "AbekasA84" } },
+  { 0xf012, { "Abekas", "AbekasA82" } },
+  { 0xf013, { "", "VPR 2/300" } },
+  { 0xf014, { "", "VPR-305" } },
+  { 0xf015, { "Pioneer", "VDR-1000A" } },
+  { 0xf016, { "Pioneer", "VDR-V100" } },
+  { 0xf017, { "Pioneer", "VDR-V100" } },
+  { 0xf018, { "Pioneer", "VDR-V100" } },
+  { 0xf019, { "Panasonic", "AJ-D350" } },
+  { 0xf01c, { "", "DA-60" } },
+  { 0xf01e, { "SONY", "HD-1200" } },
+  { 0xf01f, { "JVC", "BR-S822E" } },
```

Comment:
noting that i had it as "BR-S822U", possible one code matches more than one machine so perhaps we need to associate the code with >1 match. Same for `0x8017`.